benchmarks/bvh_driver/benchmark_registration.hpp

```diff
@@ -12,6 +12,7 @@
 #ifndef BENCHMARK_REGISTRATION_HPP
 #define BENCHMARK_REGISTRATION_HPP
 
+#include <ArborX_CrsGraphWrapper.hpp>
```

Comment:
No, the `query` free function needs to be available when including the regular `<ArborX_LinearBVH.hpp>` header file

Reply:
Good point.